eva_cttv_pipeline/clinvar_to_evidence_strings.py

```diff
@@ -193,98 +191,122 @@ def clinvar_to_evidence_strings(allowed_clinical_significance, mappings, json_fi
-
-        report.counters["n_multiple_allele_origin"] += (len(clinvar_record.allele_origins) > 1)
-        if set(clinvar_record.allele_origins).difference({"biparental", "de novo", "germline",
-                                                            "inherited", "maternal",
-                                                            "not applicable", "not provided",
-                                                            "paternal", "uniparental", "unknown"}):
-            report.counters["n_records_no_recognised_allele_origin"] += 1
-
-        traits = create_traits(clinvar_record.traits, mappings.trait_2_efo, report)
-
-        converted_allele_origins = convert_allele_origins(clinvar_record.allele_origins)
-
-        for ensembl_gene_id, trait, allele_origin \
-                in itertools.product(clinvar_record.consequence_type.ensembl_gene_ids, traits, converted_allele_origins):
-
-            if allele_origin not in ('germline', 'somatic'):
-                report.n_unrecognised_allele_origin[allele_origin] += 1
-                continue
-
-            if allele_origin == 'germline':
-                evidence_string = evidence_strings.CTTVGeneticsEvidenceString(clinvar_record,
-                                                                              report,
-                                                                              trait,
-                                                                              ensembl_gene_id,
-                                                                              cellbase_record)
-            elif allele_origin == 'somatic':
-                evidence_string = evidence_strings.CTTVSomaticEvidenceString(clinvar_record,
-                                                                             report,
-                                                                             trait,
-                                                                             ensembl_gene_id)
-            report.add_evidence_string(evidence_string, clinvar_record, trait, ensembl_gene_id)
-            report.evidence_list.append([clinvar_record.accession,
-                                         clinvar_record.rs,
-                                         trait.clinvar_name,
-                                         trait.ontology_id])
-            report.counters["n_valid_rs_and_nsv"] += (clinvar_record.nsv is not None)
-            report.traits.add(trait.ontology_id)
-            report.ensembl_gene_id_uris.add(evidence_strings.get_ensembl_gene_id_uri(
-                ensembl_gene_id))
-
-            n_ev_strings_per_record += 1
-
-        if n_ev_strings_per_record > 0:
-            report.counters["n_processed_clinvar_records"] += 1
-            if n_ev_strings_per_record > 1:
-                report.counters["n_multiple_evidence_strings"] += 1
+        clinvar_record = clinvar.ClinvarRecord(cellbase_record['clinvarSet'])
+
+        for crm in clinvar_record.measures:
```

Comment:
Could `crm` be renamed as `clinvar_record_measure` to keep consistency with the `clinvar_record` variable?